src/main/java/org/microg/nlp/backend/ichnaea/BackendService.java

```diff
@@ -44,10 +44,11 @@
     private static final String TAG = "IchnaeaBackendService";
     private static final String SERVICE_URL = "https://location.services.mozilla.com/v1/geolocate?key=%s";
     private static final String API_KEY = "068ab754-c06b-473d-a1e5-60e7b1a2eb77";
-    private static final long RATE_LIMIT_MS_FLOOR = 10000;
+    private static final long RATE_LIMIT_MS_FLOOR = 60000;
+    private static final long RATE_LIMIT_MS_PADDING = 10000;
     private static final String PROVIDER = "ichnaea";
 
-    private long EXP_BACKOFF_RATE = 1;
+    private long EXP_BACKOFF_FACTOR = 0;
```

Comment:
EXP_BACKOFF_FACTOR isn't a constant so as mar-v-in (lowercase starting camel case).

Just a personal note ( I always think about phones with low memory ;) ), does it really need to be long for just 1024 values?
Similare note for RATE_LIMIT_MS_FLOOR and RATE_LIMIT_MS_PADDING.